src/airport.js

```diff
@@ -1 +1,41 @@
+const airport = {
+    //const as in 1 airport so that wont change 
+    itemsInAirport: [],
+    defCapacity: 40,
 
+    maxCapacity(newCapacity) {
```

Comment:
i should have used more descriptive name to clarify the purpose of the method so its easier to read and understand (updateCapacity,  setCapacity)

Reply:
use syntax correctly on next challenge